gapipy/resources/base.py

```diff
@@ -78,7 +79,8 @@ def _update(self, partial=False):
 
         data = self.to_dict()
         if partial:
-            data = {k: v for k, v in data.iteritems() if self._raw_data.get(k) != v}
+            # .items isn't effecient in Python 2
+            data = {k: v for k, v in data.items() if self._raw_data.get(k) != v}
```

Comment:
Could this result in a `too many values to unpack` in py2 if data is a large dict, or does the call to `.items()` prevent that from happening?